zad6/Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '2.5.3'
+ruby '2.5.1'
```

Comment:
`ruby 2.5.3` był jak najbardziej ok, nie trzeba było zmieniać wersji 😉 Żeby zainstalować brakującą wersję wystarczy puścić `rvm install 2.5.3`(zakładając, że używamy `rvm`, ale z tego co to było narzędzie pokazywane na wykładzie).